.circleci/config.yml

```diff
@@ -859,6 +859,43 @@ jobs:
             # Upload to GitHub
             ./ghr -replace ${CIRCLE_TAG} target/wheels
 
+  pypi-linux-aarch64-release:
+    docker:
+      - image: cimg/python:3.13
+    steps:
+      - install-rustup
+      - setup-rust-toolchain
+      - checkout
+      - run:
+          name: Setup Python env
+          command: |
+            make setup-python
+            .venv3.13/bin/pip install ziglang
+      - run:
+          name: Install aarch64-linux target
+          command: |
+            rustup target add aarch64-unknown-linux-gnu
+      - run:
+          name: Build Python package
+          command: |
+            # We need a binary with debug symbols, so uniffi-bindgen can extract data
+            cargo build -p glean-bundle
+
+            . .venv3.13/bin/activate
+            make build-python-wheel GLEAN_BUILD_TARGET=aarch64-unknown-linux-gnu GLEAN_BUILD_EXTRA="--zig"
+      - run:
+          name: Upload Linux wheel
+          command: |
+            # Requires that the TWINE_USERNAME and TWINE_PASSWORD environment
+            # variables are configured in CircleCI's environment variables.
+            .venv3.13/bin/python3 -m twine upload target/wheels/*
```

Comment:
Aren't we using `uv` now?

Reply:
[not for Glean (yet)](https://github.com/mozilla/glean/blob/01c0f74fac8c253e7b2212325acdf1245ae67004/.circleci/config.yml#L823).